alembic/versions/3eb8e3fa4537_updated_subscriptions_with_user_missed_.py

```diff
@@ -0,0 +1,57 @@
+"""Updated subscriptions with user_missed_call event
+
+Revision ID: 3eb8e3fa4537
+Revises: 911ad0861ef5
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '3eb8e3fa4537'
+down_revision = '911ad0861ef5'
+
+import sqlalchemy as sa
+from sqlalchemy.dialects.postgresql import insert
+
+from alembic import op
+
+subscription_table = sa.table(
+    'webhookd_subscription',
+    sa.column('uuid'),
+    sa.column('service'),
+)
+
+subscription_event_table = sa.table(
+    'webhookd_subscription_event',
+    sa.column('subscription_uuid'),
+    sa.column('event_name'),
+)
+
+
+def upgrade():
+    mobile_subscriptions = sa.select(
+        [
+            subscription_table.c.uuid.label('subscription_uuid'),
+            sa.literal('user_missed_call').label('event_name'),
+        ]
+    ).where(subscription_table.c.service == 'mobile')
```

Comment:
Should add a condition to prevent modifying all mobile subscription as we can create custom mobile subscriptions.  Perhaps using metadata['mobile']?